app/code/Magento/User/etc/di.xml

```diff
@@ -5,15 +5,19 @@
  * See COPYING.txt for license details.
  */
 -->
-<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <type name="Magento\Authorization\Model\Role">
         <plugin name="updateRoleUsersAcl" type="Magento\User\Model\Plugin\AuthorizationRole" sortOrder="20"/>
     </type>
     <preference for="Magento\User\Api\Data\UserInterface" type="Magento\User\Model\User" />
     <type name="Magento\Framework\Console\CommandListInterface">
         <arguments>
             <argument name="commands" xsi:type="array">
-                <item name="UnlockAdminAccountCommand" xsi:type="object">Magento\User\Console\UnlockAdminAccountCommand</item>
+                <item name="UnlockAdminAccountCommand"
+                      xsi:type="object">Magento\User\Console\UnlockAdminAccountCommand</item>
+                <item name="assignAdminToRoleCommand"
+                      xsi:type="object">Magento\User\Console\AssignAdminToRoleCommand</item>
```

Comment:
You don't need to break the lines.
Characters limit in a row affects only PHP.

Reply:
It's not forbidden, And I like it more.

Reply:
OK.